docs/agent-api.md

```
@@ -239,11 +271,101 @@ Live tool registry + confirmation wiring: `list()`, `setEnabled(names)`,
+  includeResources: true,
+});
+for (const server of detail.servers) {
+  console.log(server.name, server.authenticated, server.tools?.length);
+}
+```
+
+> **Refresh parity.** `agent.mcp.refresh(server?)` restarts the target server
+> (or all servers when called with no argument) **and** re-publishes its tool
+> declarations (`setTools` parity), so the live tool list tracks server
+> restarts without a separate manual publish step.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)